### PR TITLE
Merge nonode branch into master by adding the env variable PHP_BUILDPACK_NO_NODE

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ Install CLI tool → http://cli.scalingo.com
 If you have `package.json` file in your repository, Node.JS will be installed automatically
 and dependencies will be installed according to the `package.json` file.
 
+You can disable this feature by defining the environment variable `PHP_BUILDPACK_NO_NODE=true`
```
Comment (EtienneM) [inline README.md]:
```suggestion
You can disable this feature by defining the environment variable `PHP_BUILDPACK_NO_NODE=true`.
```

Is it possible to explain the cases where one would need to disable this feature?

**Comment by Soulou** [comment]
Review done

**Comment by Soulou** [comment]
> Is it possible to get from the API the list of apps which use this branch of the buildpack?

It's unfortunately not possible today

**Comment by Soulou** [comment]
We could break the build with #nonode and clearly display that the branch has been deprecated and that #master should be used with the env var.

**Comment by EtienneM** [comment]
> We could break the build with #nonode and clearly display that the branch has been deprecated and that #master should be used with the env var.

Yes I really like this idea! Can you quickly implement this? Then LGTM

File: README.md [hunk 2]
```diff
@@ -29,6 +29,12 @@ Install CLI tool → http://cli.scalingo.com
 If you have `package.json` file in your repository, Node.JS will be installed automatically
 and dependencies will be installed according to the `package.json` file.
 
+You can disable this feature by defining the environment variable `PHP_BUILDPACK_NO_NODE=true`.
+
+You might want to disable it when you want a fined grained management of the Node.js part by using
+the official [Node.js buildpack](https://github.com/Scalingo/nodejs-buildpack) in combinaision with this
+buildpack thanks to the [Multi buildpack](https://doc.scalingo.com/platform/deployment/buildpacks/multi)
```
Comment (EtienneM) [inline README.md]:
```suggestion
buildpack thanks to the [Multi buildpack](https://doc.scalingo.com/platform/deployment/buildpacks/multi).
```

File: README.md [hunk 3]
```diff
@@ -29,6 +29,12 @@ Install CLI tool → http://cli.scalingo.com
 If you have `package.json` file in your repository, Node.JS will be installed automatically
 and dependencies will be installed according to the `package.json` file.
 
+You can disable this feature by defining the environment variable `PHP_BUILDPACK_NO_NODE=true`.
+
+You might want to disable it when you want a fined grained management of the Node.js part by using
+the official [Node.js buildpack](https://github.com/Scalingo/nodejs-buildpack) in combinaision with this
```
Comment (EtienneM) [inline README.md]:
```suggestion
the official [Node.js buildpack](https://github.com/Scalingo/nodejs-buildpack) in combination with this
```

**Comment by Soulou** [comment]
But I'm really not sure we want to break the builds actually.... but I can add a big warning, it won't be in this PR as it's another branch

**Comment by EtienneM** [comment]
If we don't break the build, I'm 99% sure that nobody will read the big warning ^^

**Comment by EtienneM** [comment]
I created an issue (#91) for the deprecation of the `nonode` branch.

**Pull request merged**